bin/_docker.sh

```diff
@@ -1,20 +1,21 @@
 set -eu
 
-bindir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+#bindir=$( cd "${BASH_SOURCE[0]%/*}" && pwd )
```

Comment:
did you mean to remove this commented out line?

Reply:
No, I did not, thanks for observing it. Funny enough, my test builds as well as the CI did work (since the bindir is set in the scripts that source _docker.sh). But it was not intended to be commented out. I will fix this.